Configs/.local/lib/hyde/cyclewallpaperandtheme.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env sh
+
```

Comment:
Use bash in your shebang if the script is not POSIX (sh) at all, so the linters won't scream. 
```suggestion
#!/usr/bin/env bash

```